api_docs/configuring-python-bindings.md

```diff
@@ -13,7 +13,53 @@ settings. There are two ways to achieve that:
    [environment variables](https://en.wikipedia.org/wiki/Environment_variable)
    set up in your host machine.
 
-A `.zuliprc` file is a plain text document that looks like this:
+## Download a `.zuliprc` file
+
+{start_tabs}
+
+{tab|via-active-bots}
+
+{settings_tab|your-bots}
+
+1. Click the **download** (<i class="fa fa-download"></i>) icon on the profile
+   card of the desired bot to download the bot's `zuliprc` file.
+
+!!! tip ""
+
+    If you save this file as `.zuliprc` in your home directory (`~/`), the
+    Python API bindings will automatically read it in.
+
+!!! warn ""
+
+    Anyone with a bot's API key can impersonate the bot, so be careful with it!
+
+{tab|via-account-privacy}
+
+{settings_tab|account-and-privacy}
+
+1. Under **API key**, click **Manage your API key**.
+
+1. Enter your password, and click **Get API key**. If you don't know your
+   password, click **reset it** and follow the
+   instructions from there.
+
+1. Click **Download .zuliprc** to download your `zuliprc` file.
+
+!!! tip ""
+
+    If you save this file as `.zuliprc` in your home directory (`~/`), the
+    Python API bindings will automatically read it in.
+
+!!! warn ""
+
+    Anyone with your API key can impersonate you, so be doubly careful with it.
+
+{end_tabs}
+
+## Configuration keys and environment variables
+
+A `.zuliprc` file is a plain text document that contains key-value pairs
+formatted like this:
```

Comment:
Technically, the format is an [INI file](https://en.wikipedia.org/wiki/INI_file), I think.

Reply:
Hmm, maybe we should say something like:

> `.zuliprc` is a configuration file written in the [INI file format](https://en.wikipedia.org/wiki/INI_file), which contains key-value pairs as shown in the following example: